src/booleans.js

```diff
@@ -1,61 +1,61 @@
 function negate(a) {
-  // your code here
+  return !a;
 };
 
 function both(a, b) {
-  // your code here
+  return (a&&b);
 };
 
 function either(a, b) {
-  // your code here
+  return (a||b);
 };
 
 function none(a, b) {
-  // your code here
+  return !(a||b);
 };
 
 function one(a, b) {
-  // your code here
+  return (a&&!b) || (!a&&b);
```

Comment:
As above:
```
  return (a && !b) || (!a && b);
```
The rest of the function here should be formatted to fit with this style